docs/appendices/release-notes/unreleased.rst

```diff
@@ -202,6 +202,13 @@ Data Types
 Administration and Operations
 -----------------------------
 
+- Added optimizer rules for reordering of joins for hash and nested-loop joins.
+  This allows now to control the join-reordering and disable it if desired with
+  the following session settings:
+
+  SET optimizer_reorder_hash_join = false
+  SET optimizer_reorder_nested_loop_join = false
```

Comment:
I think we should mention that these are experimental settings and such may change in future.

Reply:
+1